Doc/library/pdb.rst

```diff
@@ -118,11 +118,15 @@ slightly different way:
    is entered.
 
 
-.. function:: set_trace()
+.. function:: set_trace(*, header=None)
 
-   Enter the debugger at the calling stack frame.  This is useful to hard-code a
-   breakpoint at a given point in a program, even if the code is not otherwise
-   being debugged (e.g. when an assertion fails).
+   Enter the debugger at the calling stack frame.  This is useful to hard-code
+   a breakpoint at a given point in a program, even if the code is not
+   otherwise being debugged (e.g. when an assertion fails).  If given,
+   ``header`` is printed to the console just before debugging begins.
+
+   .. versionadded:: 3.7
```

Comment:
I never remember if versionadded in a function will cause sphinx to report that the whole function is new, or will display the text that you wrote.

Reply:
Thanks for the review!  It displays the text; from a local build:

```
New in version 3.7: The keyword-only argument header.
```